integration-tests/tzip16-originate-wallets-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts

```diff
@@ -114,7 +114,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
             done();
         });
 
-        it('Fetch the metadata which contains emoji of the wallet api contract', async (done) => {
+        it('Verify that the metadata for the contract which contains emoji of the contract can be fetched', async (done) => {
```

Comment:
Verify that the metadata for the contract which contains emoji ~of the contract~ can be fetched

Reply:
Changed to 
'Verify that the metadata for the contract which contains emoji can be fetched'